### DISCUSSION TITLE
Lost BG readings in NightScout

### DISCUSSION THREAD

**Discussion by data4tim** [NightscoutFoundation/xDrip#3346] [Q&A]
Nightscout was running fine but now I've lost my BG readings. I still get the Loop info pulled in but no current BG. I am running via Google Cloud. I updated Nightscout and reentered my freeDNS user and password and there are now no errors on the status but still no BG readings in Nightscout. Anyone know how I can fix this?

**Comment by Navid200** [comment]
Restart the server: https://navid200.github.io/xDrip/docs/Nightscout/Restart.html

Then, post a screenshot of the status page: https://navid200.github.io/xDrip/docs/Nightscout/Status.html

**Reply by data4tim** [reply]
Thanks but no need, problem already solved. I found I simply needed to turn 'upload readings' back on in Dexcom settings of Loop. This needs to be done each time a new transmitter is used (still using G6).
